dns-production-0/Makefile

```diff
@@ -0,0 +1,20 @@
+ENV := dns
+ENV_SHORT := production
+INDEX := 0
+
+include $(shell git rev-parse --show-toplevel)/terraform-common.mk
+
+PROD_TF_VERSION := v0.11.2
```

Comment:
We've pinned terraform version to v0.9.11 because of the vpshere provider changes in the 0.10+ version change.

Reply:
err. pinned for macstadium

Reply:
Hardcoding the version might conflict with (bin/ensure-terraform)[https://github.com/travis-ci/terraform-config/blob/master/bin/ensure-terraform#L13] arch check ?

Reply:
@Lyoness this version pin will only apply to this graph/dir :+1: